src/helpers/theme_provider.tsx

```diff
@@ -48,13 +48,9 @@ export const ThemeProvider = ({ theme, children }: ThemeProviderProps) => {
 
 export const withThemeProvider = (
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  Story: React.ComponentType<StoryContext<ReactFramework, any>>,
+  Story: React.FunctionComponent<StoryContext<ReactFramework, any>>,
```

Comment:
It’s a regular parameter, so it should have a lower-case name.
```suggestion
  story: React.FunctionComponent<StoryContext<ReactFramework, any>>,
```